src/ShopBundle/objects/TCMSSmartURLHandler/TCMSSmartURLHandler_ShopPayPalAPI.class.php

```diff
@@ -14,7 +14,10 @@
 
 /**
  * takes paypal return urls and mapps them to chameleon urls.
-/**/
+ *
+ * @psalm-suppress UndefinedPropertyAssignment
+ * @FIXME Writing data into `$OURLData` when there is no magic `__set` method for them defined.
```

Comment:
In this class, there is a lot of writing into 'magic' properties of `TCMSSmartURLData` - however, those properties are not writable.